core/domain/services/assets/EspressoEditorAssetManager.php

```diff
@@ -48,7 +48,10 @@ private function registerJavascript()
                 ReactAssetManager::JS_HANDLE_REACT,
                 ReactAssetManager::JS_HANDLE_REACT_DOM,
                 CoreAssetManager::JS_HANDLE_JS_CORE,
+                'wp-components',
```

Comment:
This is for wpPluginsPage

Reply:
`EspressoEditorAssetManager` doesn't load on the WP Plugins page, so these will need to be added to `WordpressPluginsPageAssetManager` for the time being. I'll have to kill off a bunch of these asset managers though since we are using a single entry point now.

Reply:
Oh yes. Thanks 👍